e2etest/get-performance-model-table.py

```diff
@@ -58,9 +58,38 @@ def flatten_performance_models(models):
         x["data_mode"], x["data_rate"]))
     return models_list
 
+def get_benchmark_data():
+    """
+    Splits models by test
+    """
+    benchmark_data = []
```

Comment:
IMO, instead of using only [], using list(dict) would have more clarification in defining what is added into the list.